File: src/Java/task_2/pangram.java
import java.util.Scanner;
import java.util.Set;
import java.util.HashSet;

public class pangram {

    public static boolean isPangram(String sentence) {

        Set<Character> letters = new HashSet<>();

        for (char ch : sentence.toLowerCase().toCharArray()) {

            if (Character.isLetter(ch)) {
                letters.add(ch);
            }
        }

        return letters.size() == 26;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String sentence = scanner.nextLine();
        System.out.println(isPangram(sentence));
        scanner.close();
    }
}

File: src/Kotlin/task_2/pangram.kt
fun isPangram(sentence: String): Boolean {
    // Создаем множество для хранения уникальных символов
    val letters = mutableSetOf<Char>()

    // Проходим по каждому символу в строке
    for (char in sentence) {
        // Если символ является буквой, добавляем его в множество в нижнем регистре
        if (char.isLetter()) {
            letters.add(char.toLowerCase())
        }
    }

    // Панграмма должна содержать все 26 букв английского алфавита
    return letters.size == 26
}

fun main() {
    println()
    val input = readLine() ?: ""

    // Выводим результат проверки
    println(isPangram(input))
}